test_scripts/Defects/5_0/2454_SDL_does_not_clear_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2454
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
+-- Perform MASTER_RESET
+
+-- Expected Behavior:
+-- All SDL data are cleaned and reset.
+---------------------------------------------------------------------------------------------------
+
+-- For useing this script you must set actual path to binary file directory of SDL to 'config.pathToSDL' in config.lua file.
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local common = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local config = require("modules/config")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local function HMISendToSDL_MASTER_RESET()
+	common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications",
+		  { reason = "MASTER_RESET" })
+	common.getMobileSession(1):ExpectNotification("OnAppInterfaceUnregistered", 
+          { reason = "MASTER_RESET" })
+          :Times(1)
+	common.getMobileSession(2):ExpectNotification("OnAppInterfaceUnregistered", 
```

Comment:
please remove whitespase in the end of line

Reply:
@GetmanetsIrina Removed in bf1c6b75fe5f4e1f3d891034a6faf412bcf250e1.